mono/metadata/threads.c

```diff
@@ -6350,8 +6326,14 @@ summarizer_supervisor_start (SummarizerSupervisorState *state)
 	g_assert_not_reached ();
 #endif
 
-	if (pid != 0)
+	if (pid != 0) {
 		state->supervisor_pid = pid;
+		char pid_str[20]; // pid is a uint64_t, 20 digits max in decimal form
+		sprintf (pid_str, "%llu", (uint64_t)state->pid);
+		const char *const args[] = { "mono-hang-watchdog", pid_str };
+		execvp (args[0], (char * const*)args); // run 'mono-hang-watchdog [pid]'
```

Comment:
Is it reasonable to rely on `$PATH` to find `mono-hang-watchdog`?  Maybe we should make some path relative to the `mono` binary?  (`../libexec/mono-hang-watchdog` perhaps?)